lib/ddtrace/contrib/resque/patcher.rb

```diff
@@ -17,11 +17,11 @@ def target_version
 
         def patch
           require_relative 'resque_job'
-          get_option(:workers).each { |worker| worker.extend(ResqueJob) }
-        end
 
-        def get_option(option)
-          Datadog.configuration[:resque].get_option(option)
+          ::Resque::Job.send(:prepend, Resque::Job)
+
+          workers = Datadog.configuration[:resque][:workers] || []
+          workers.each { |worker| worker.extend(ResqueJob) }
```

Comment:
Minor: I was thinking we could instead tweak `Datadog::Contrib::Resque::Job` to instrument workers in the list "just-in-time", similar to how we do it when workers is `nil`. This way, we would have almost the same behavior between both options, instead of the current setup where the extend happens at different times based on the configuration. (And we could remove these two lines entirely)

Reply:
I didn't want to touch these lines, because this behaviour (explicit configuration) is going to be deprecate anyway. It's a low risk change, but given the low value, I thought leaving it as it, and simply removing in the future was the best thing to to day.